configure

```diff
@@ -1,8 +1,8 @@
 #! /bin/sh
 # Guess values for system-dependent variables and create Makefiles.
-# Generated by GNU Autoconf 2.69 for BOUT++ BOUT_VERSION.
+# Generated by GNU Autoconf 2.69 for BOUT++ v3.0-852-g2cd825c.
 #
-# Report bugs to <bd512@york.ac.uk>.
+# Report bugs to <benjamin.dudson@york.ac.uk>.
```

Comment:
I think detailed instructions for reporting bugs should go on the website (along with many other things)